pages/index.js

```diff
@@ -33,10 +34,14 @@ export default function HomePage({
   setSearchResults,
   handleAddEntry,
   handleActivePage,
+  handleActiveUser,
 }) {
+  const { data: session } = useSession();
+
   useEffect(() => {
     handleActivePage("home");
-  }, [handleActivePage]);
+    handleActiveUser(session?.user._id);
+  }, [handleActivePage, handleActiveUser, session]);
```

Comment:
instead of performing this on each page individually, you can do this once in the `_app.js`

Reply:
I moved that to the navigation itself using router. I don't need it for anything else, I think. :)